src/travel-details-screens/TripDetailsScreenComponent.tsx

```diff
@@ -151,73 +151,155 @@ export const TripDetailsScreenComponent = ({
 };
 
 function useGetTicketInfoFromTrip(tripPattern: TripPattern) {
```

Comment:
Is `Get` needed in the name when it's a hook? I see that you haven't touched the name, but figured I should ask anyways. 😀

Reply:
Good suggestion! I've updated the name to `useTicketInfoFromTrip` 😊